### ISSUE TITLE
Duplicate IDs across the examples of the Design System

### ISSUE THREAD

**Issue by emilymaylawes** [ONSdigital/design-system#2731]
Across all component and patterns pages of the design system, where component examples have been used, there are quite often duplicate IDs for different elements within the examples, when compared to other examples on the same page.

This happens across most pages, so it is harder to pinpoint the exact location. This has been discovered by a quick automated audit using both SiteImprove and AXE Dev Tools.

Good examples of this are on the [Checkbox ](https://service-manual.ons.gov.uk/design-system/components/checkboxes) pattern, where the same checkbox text across different examples likely has the same ID (`mobile-phone` as an example).

This also happens as an example on the [Card ](https://service-manual.ons.gov.uk/design-system/components/card) page where within each example 'Nunjucks macro options' tab, there is a heading "Image" which has the ID `#image` which is repeated across the page on all examples.

[WCAG 2.2](https://www.w3.org/TR/WCAG22/) is removing 4.1.3. Parsing from their guidelines so going forwards this will no longer be a failure, but for WCAG 2.1 which we are still aligning to, and for backwards compatibility, it is recommended to review all pages and ensure there are no duplicate IDs. It is also best coding practice. 

**This is a bug that will be placed in the accessibility statement.**
